third_party/move/move-compiler-v2/tests/variable-coalescing/consume_2.move

```diff
@@ -0,0 +1,21 @@
+module 0xc0ffee::m {
+    fun consume(_x: u64) {}
+
+    public fun test1(x: u64) {
+        let y = move x;
+        consume(y);
+        consume(y);
```

Comment:
shouldn't we see an error here?

Reply:
I do not see any reason to report an error here (note: integers have the copy ability).